### ISSUE TITLE
show more info when it's not my turn

### ISSUE THREAD

**Issue by mohrm** [mohrm/umklapp_site#66]
@jgf had the following idea: When it's not my turn, why not show the story parts I have contributed up until now and also the story parts on which I reacted.

In essence, we would display everything which I would remember if I had perfect memory. This would be beneficial especially for long-running stories with many contributors where I don't neccessarily remember what I have written so far.

We could make this optional, i.e. present the user a button-like entity which says 'show history.

What do you think?

**Comment by ratefuchs** [comment]
If we implement this, there would be really no reason to restrict it so that we only show this if it is not my turn. I'd rather have that feature when it **is my turn** than when it is not (both is fine, though).

Still, there is a decision to make for two teller stories: We would show the whole story in this case. I'd leave that behavior up for discussion.

**Comment by mohrm** [comment]
> If we implement this, there would be really no reason to restrict it so that we only show this if it is not my turn. I'd rather have that feature when it is my turn than when it is not (both is fine, though).

Yes, you're right. If I think about this, I would like to always show it, i.e. also when it is my turn, of course.

> Still, there is a decision to make for two teller stories: We would show the whole story in this case. I'd leave that behavior up for discussion.

If we make the feature optional, there is no need to differentiate because the user can then choose to spoil the game.

**Comment by nomeata** [comment]
As much as I wanted it at times, I think this feature goes agains the spirit of the game.

**Comment by mohrm** [comment]
> As much as I wanted it at times, I think this feature goes agains the spirit of the game.

I know that this is not really like the game works in reality. But a substantial difference to the game in reality also is that the virtual game runs over a larger period of time. In reality, a story is written in half an hour and not half a month. Also, in reality you have have less players. So, in reality, you most likely won't be in a situation where you can't remember what you wrote last time.

I'd also like to stress that I do _not_ want to reveal everything that has been written up until now but only the information which I would get anyway if I just copy what I see and what I write every time it's my turn. So, for a user with perfect memory (or someone who copies a lot), there would be no information gain.
Plus, I think we should leave it up to the users, whether they want this information or not.

**Comment by lohner** [comment]
Why not leave it (also) open to the story creator if he wants that feature for his story and make it optional on a per story basis. I could imagine, that it is very useful for large stories with 10+ tellers and rather long turnaround times.
